packages/api/tests/endpoints/collections/delete-collection.js

```diff
@@ -83,7 +82,6 @@ test.before(async (t) => {
 
   t.context.collectionModel = new models.Collection({ tableName: process.env.CollectionsTable });
   await t.context.collectionModel.createTable();
```

Comment:
I think, you can delete both os these lines, and the cleanup on L140 if you update the createCollectionTestRecords function to check for a collectionModel before creating a model
```
  if (collectionModel) {
    await collectionModel.create(originalCollection);
  }
```
That way there's no api.model collections in these tests at all.

I think that's better, but you can disagree

Reply:
I think it's better because it's what I did in my removal PR . #2686 remove dynamo from provider delete.

Reply:
Thanks for the feedback! I took your suggestions and updated the test setup and the createCollectionTestRecords function [`8f7216b` (#2694)](https://github.com/nasa/cumulus/pull/2694/commits/8f7216bc6b4ffa767d2d9954ef9c1f5b796b391a)